ini/stm32f1.ini

```diff
@@ -69,7 +69,7 @@ board_upload.offset_address = 0x08007000
 [env:STM32F103RC_btt_USB]
 extends           = env:STM32F103RC_btt
 platform          = ${common_stm32.platform}
-platform_packages = framework-arduinoststm32@https://github.com/rhapsodyv/Arduino_Core_STM32/archive/usb-host-msc-cdc-msc.zip
+platform_packages = framework-arduinoststm32@https://github.com/rhapsodyv/Arduino_Core_STM32/archive/refs/heads/usb-host-msc-cdc-msc-2.zip
```

Comment:
If you put a tag on this commit in your repository, or copy this commit into a separate branch, then you can drop "`refs/heads`" and use the name of the tag or branch instead.